### PR TITLE
Updated Code of Conduct

### PR DESCRIPTION
**Pull request by ZeCarlosCoutinho**
- Translated the code of conduct to PT.
- Adds the email for cases of code violation

### CODE REVIEW

**Comment by ZeCarlosCoutinho** [comment]
There are some differences between the CoC in the code and the CoC in this document:
https://docs.google.com/document/d/1en4OmzPVT4kB0sbrl4roccluI9gqvt7Aox3YGNNCooY/edit

If the document is the most updated one, then I will need to change the markdown text before this is merged.

File: app/views/home/code_of_conduct.pt.html.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
 # Código de conduta
 
-TODO
+O objetivo principal de [mentor.alumniei.pt](/) é ser inclusivo para toda a comunidade de alumni e estudantes de cursos da FEUP/LEIC, com os passados mais variados e diversos possível.  O nosso compromisso é proporcionar um ambiente amigável, seguro e acolhedor para todos, independentemente de género, orientação sexual, habilidade, etnicidade, estatuto socio-económico e religião.
+
+Este código de conduta delineia quais os comportamentos que esperamos que a comunidade cumpra, bem como as consequências de comportamento inaceitável.
+
+Se fores um alumni ou estudante FEUP/LEIC, nós convidamos-te a participar nesta plataforma para nos ajudar a create um espaço seguro e positivo para partilhar experiências, conhecimento e questões.
```
Comment (josealeixopc) [inline app/views/home/code_of_conduct.pt.html.md:7]: create > criar

File: app/views/home/code_of_conduct.pt.html.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+- Respeitar a confidencialidade sobre elementos e informação partilhada nesta plataforma.
+- Respeitar a comunidade. Não proliferes comportamentos e discurso humilhantes, discriminatórios ou de assédio.
+- Reportar membros que vão contra o nosso código de conduta.
+
+Alguns exemplos de comportamentos que não serão tolerados:
+- Proliferar piadas e linguagem sexista, racista, homofóbica, transfóbica, relativas a incapacidade/inaptidão ou de outro modo discriminatórias.
+- Postar e/ou mostrar material sexualmente explícito, violento ou inapropriado.
+- Desrespeitar de qualquer modo um membro da comunidade.
+
+Se um mentor ou estudante demonstrar comportamento perturbador, o staff pode tomar qualquer ação que achar apropriada, includindo avisar o transgressor ou bani-lo da plataforma.
```
Comment (josealeixopc) [inline app/views/home/code_of_conduct.pt.html.md:23]: includindo > incluindo

**Review by josealeixopc — APPROVED** [review]
Looks ok to me! I can change between the codes of conduct by changing the language. I've highlighted some typs on the portuguese version.

An extra observation on the `code_of_conduct.en.html.md` file, line 20: "students time" > "students' time".

EDIT: Regarding the CoC on Google Docs, I'd suggest the following: Keep the current code of conduct (I feel that "setting in stone" a ban period may be harmful. Best to handle each case individually, in case of bad conduct) and only change the following lines:

"- Respect students time by keeping your profile up to date and letting us know when you are not available to mentor students."
to
"- Keep your data up to date. Turn your profile as inactive when you are not available to mentor students." (sound less demanding)

**Pull request merged**